CHANGES.md

```diff
@@ -1,5 +1,6 @@
-1.1.0
+1.1.0 (2015-01-12)
```

Comment:
You can leave this line as it is, we only add the date there when we actually make the release.